src/main/java/abel/ArrayList.java

```diff
@@ -0,0 +1,86 @@
+package abel;
+
+import java.util.Arrays;
+
+/**
+ * Class that implements a simple Array List.
+ *
+ * @param <T> Generic Object.
+ */
+public class ArrayList<T> implements List {
```

Comment:
```suggestion
public class ArrayList<T> implements List<T> {
```
Don't forget to pass the T